src/middleware/changeURL.js

```diff
@@ -82,6 +82,14 @@ export const changeURLMiddleware = (store) => (next) => (action) => {
     case types.NEW_COLORS:
       query.c = action.colorBy === state.controls.defaults.colorBy ? undefined : action.colorBy;
       break;
+    case types.SET_FOCUS:
+      if (action.focus === null) {
+        delete query.focus;
+      }
+      else {
+        query.focus = action.focus;
+      }
```

Comment:
Small note, but this used to be one line:

https://github.com/nextstrain/auspice/blob/7ef2674324c50e512780807b7da2f6dad387f2e9/src/middleware/changeURL.js#L86

I've updated it to use `delete` since that's what's used in `modifyStateViaURLQuery` and is more readable to me.